resources/providers/azurelib/inventory/provider.go

```diff
@@ -28,6 +28,7 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armsubscriptions"
 	"github.com/elastic/elastic-agent-libs/logp"
 	"github.com/samber/lo"
+	"golang.org/x/exp/maps"
```

Comment:
fyi [it was approved to be added to stdlib in Go 1.21](https://github.com/golang/go/issues/57436#issuecomment-1412566002)